other_processing/0.download-profiles-from-figshare/download.py

```diff
@@ -44,7 +45,11 @@ def download_sqllite_file(filename: Union[str, Path], url: str):
         All downloaded profiles will be stored in the `./data` directory
     """
 
-    print("Now downloading... {}".format(filename))
+    if isinstance(filename, str):
+        filename = Path(filename)
+
+    plate_name = filename.name
+    print("Now downloading... {}".format(plate_name))
```

Comment:
Consider using an f-string here:
```suggestion
    print(f"Now downloading... {plate_name}")
```